R/scale-colour.r

```diff
@@ -0,0 +1,46 @@
+#' Continuous colour scales
+#'
+#' Colour scales for continuous data default to the values of the
+#' `ggplot2.continuous.colour` and `ggplot2.continuous.fill` options. If these
+#' options are not present, ggplot2 will set them to `"gradient"`. See
+#' [options()] for more information.
+#' 
+#' @param ... Additional parameters passed on to the scale type
+#' @param type One of "gradient" (the default) or "viridis" indicating the
+#'   colour scale to use
+#' @seealso [scale_colour_gradient()], [scale_colour_viridis_c()],
+#'   [scale_fill_gradient()], and [scale_fill_viridis_c()]
+#' @export
+#' @rdname scale_colour_continuous
+#' @examples
+#' v <- ggplot(faithfuld, aes(waiting, eruptions, fill = density)) +
+#' geom_tile()
+#' v
+#' 
+#' v + scale_fill_continuous(type = "gradient")
+#' v + scale_fill_continuous(type = "viridis")
+#' 
+#' # The above are equivalent to
+#' v + scale_fill_gradient()
+#' v + scale_fill_viridis_c()
+scale_colour_continuous <- function(...,
```

Comment:
I now think it's probably better form to use the second argument to `getOption()`, rather than setting on load.